gulpfile.js

```diff
@@ -62,8 +62,7 @@ var searchScripts = baseScripts
         'src/infinite-scroll.js',
         'src/show-more-questions.js',
         'src/mobile-menus.js',
-        'src/search.js',
-        'src/api/odn-api.js']);
```

Comment:
Is this intentionally removed?

Reply:
Hmmm, looks like the file doesn't exist.

Reply:
See the "Security Patches" change above. Notes say:
_gulp upgraded to 4.0.2, with changes to gulpfile.js src/api/odn-api.js is a non-existent file._

Reply:
@haibeeb Are your changes safe to merge to master?